### PR TITLE
German ROM: berry.c

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by YamaArashi** [comment]
How can Travis check builds when the German Ruby won't have the correct hash for a while?

**Comment by marijnvdwerf** [comment]
It should only validate whether the German rom actually builds. Having the decompiled stuff in is easier than trying to keep it in sync.

**Comment by YamaArashi** [comment]
OK. I think we need to add a new phony target for German builds, in order to set the language properly.

File: .travis.yml [hunk 1]
```diff
@@ -28,3 +28,4 @@ script:
   - make -j2 compare_sapphire
   - make -j2 compare_sapphire_rev1
   - make -j2 compare_sapphire_rev2
+  - make -j2 pokeruby_de.gba
```
Comment (YamaArashi) [inline .travis.yml]: make -j2 ruby_de

**Pull request merged**